altair/schema/_wrappers/channel_wrappers.py

```diff
@@ -0,0 +1,257 @@
+# This file auto-generated by `generate_schema_interface.py`.
+# Do not modify this file directly.
+
+import traitlets as T
+import pandas as pd
+
+from ...utils import parse_shorthand, infer_vegalite_type
+from ...utils import INV_TYPECODE_MAP, TYPE_ABBR
+
+from .._interface import Type
+from .._interface import PositionChannelDef
+from .._interface import ChannelDefWithLegend
+from .._interface import FieldDef
+from .._interface import OrderChannelDef
+
+
+class PositionChannel(PositionChannelDef):
+    """Wrapper for Vega-Lite PositionChannelDef definition.
+    
+    Attributes
+    ----------
+    shorthand: Unicode
+        A shorthand description of the channel
+    aggregate: AggregateOp
```

Comment:
Blank lines indicate a blank attribute description. Do you think we should try to remove them?